Tasks/Sergey Yashchanka/Task_1/monthly_capitalization.py

```diff
@@ -0,0 +1,48 @@
+# Задание:
+
+# Депозит:
+# начальная сумма - 20000 BYN
+# срок - 5 лет
+# процент (годовой) - 15%
+# !ежемесячная капитализация!
+
+# Вычислить сумму на счету в конце указанного срока.
+
+# ------------------------------------------------------------------------------------------------------------------ #
+
+# Решение N_1: EASY LEVEL !!!
+
+# formula monthly capitalization
+# S = D * ((1 + (n / 100) / M) ** (T * M - 1))
+#
+# S - account balance at the end of the term!
+# D - initial deposit amount!
+# n - annual interest!
+# M - number of months in the year!
+
+
+initial_sum = 20000
+period = 5
+annual_interest = 15
+MONTHS_IN_YEARS = 12
+
+# find the total amount on the account using the formula
+total_sum = initial_sum * pow((1 + (annual_interest / 100) / MONTHS_IN_YEARS), (period * MONTHS_IN_YEARS - 1))
+print(f"Сумма на счету в конце указанного срока составит: {str(total_sum)[:5]}р {str(total_sum)[6:8]}k")
```

Comment:
it's a tricky topic, but I would say that rounding is better than string slicing

Reply:
Thank you for your comments! I will use the rounding next time !